sdk/zksend/src/wallet/index.ts

```diff
@@ -123,11 +135,41 @@ export class StashedWallet implements Wallet {
 		};
 	};
 
+	#signTransactionBlockV2: SuiSignTransactionBlockV2Method = async ({
+		transactionBlock,
+		account,
+	}) => {
+		const popup = new StashedPopup({
+			name: this.#name,
+			origin: this.#origin,
+			type: 'sign-transaction-block',
+		});
+
+		const txb = TransactionBlock.from(await transactionBlock.toJSON());
+		txb.setSenderIfNotSet(account.address);
+
+		const data = txb.serialize();
```

Comment:
Are we doing this for compat?

Reply:
yes, we need to handle the new format on the zksend side first.  Once it's updated to the new SDK version, we should be able to update to the new format here (once zksend is on the new sdk version, it should be able to parse either format with the same method)